### PR TITLE
Improve commit guidelines

### PR DESCRIPTION
**Pull request by eternoendless**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | Improved the part about commits in the contribution guidelines
| Fixed ticket? | N/A

Screenshot:
Screenshot 2022-02-03 at 17 44 58

### CODE REVIEW

File: contribute/contribution-guidelines/_index.md [hunk 1]
```diff
@@ -44,17 +44,12 @@ For more information, see [Supported branches]({{< ref "supported-branches" >}})
     ```bash
     git config --global user.name "Your Name"
     git config --global user.email yourname@yourcompany.com
-    ```
+    ```
```
Comment (NeOMakinG) [inline contribute/contribution-guidelines/_index.md]:
```suggestion
    ```
```

Well... This is the most valuable review ever

**Review by NeOMakinG — APPROVED** [review]

File: contribute/contribution-guidelines/_index.md (hunk 1 above)
Comment (eternoendless) [reply contribute/contribution-guidelines/_index.md]: Fixed

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @eternoendless

**Comment by matks** [comment]
PR for backport https://github.com/PrestaShop/docs/pull/1296

**Pull request merged**